lms/services/jwt.py

```diff
@@ -107,7 +107,7 @@ def decode_lti_token(self, id_token: str) -> dict:
 
         try:
             signing_key = self._get_jwk_client(
-                registration.key_set_url
+                self._http_service, registration.key_set_url
```

Comment:
The first failure is the first contact with the school to get the JWKs.

I don't want to integrate AIA twice so instead moved this service to use HTTPService instead of raw `requests`